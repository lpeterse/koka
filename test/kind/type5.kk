type int
type app<f,a> { con App( app: f<a>) }
type id<a> { con Id(x:a) }
alias ok = app<id,int>
