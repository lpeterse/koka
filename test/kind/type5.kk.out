kind/type5/app: (V -> V, V) -> V
kind/type5/id: V -> V
kind/type5/int: V
kind/type5/ok: V
public alias ok = app<id,int> = 1